src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -257,11 +257,9 @@ public override async ValueTask<FormulaValue> Visit(CallNode node, EvalVisitorCo
             {
                 result = await asyncFunc.InvokeAsync(args, _cancellationToken).ConfigureAwait(false);
             }
-#pragma warning disable CS0618 // Type or member is obsolete
-            else if (func is IAsyncTexlFunction2 asyncFunc2)

```

Comment:
>IAsyncTexlFunction2 [](http://example.com/codeflow?start=29&length=19)

To be safe, let's delete IAsyncTexlFunction2 in a separate PR.

Reply:
Can do that